website/assets/styles/pages/articles/basic-webinar.less

```
@@ -0,0 +1,749 @@
+    margin-bottom: 40px;
+    width: 100%;
+    padding-bottom: 56%;
+    border-radius: 12px;
+    iframe {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      border: none;
+      // border-radius: 12px;
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)